src/codeu/chat/common/BasicController.java

```diff
@@ -49,6 +49,6 @@
   //  operation is successful, a Conversation object will be returned
   //  representing the full state of the conversation on the server.
   //  Whether conversations can have the same title is undefined.
-  ConversationHeader newConversation(String title, Uuid owner);
+  ConversationHeader newConversation(String title, Uuid owner, Integer defaultByte);
```

Comment:
Likely no need for the "Integer" class here (and almost everywhere else --- except when it's the parameter of a generic as in HashMap<..., Integer>). Just a plain old "int" should suffice.